### PR TITLE
Added Issue Templates

### PR DESCRIPTION
**Pull request by adityaraute**
## Changes proposed

Added Issue templates as seen in the BioDrop repository

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by flexcommits — APPROVED** [review]
Good!!!

**Comment by adityaraute** [comment]
Bumping this PR

**Review by xkrishguptaa — CHANGES_REQUESTED** [review]
We don't need the testimonial one

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question?
+    url: https://www.biodrop.io/docs/faqs
+    about: If your question is not covered by the FAQs, please ask your question in our GitHub Discussions.
```
Comment (xkrishguptaa) [inline .github/ISSUE_TEMPLATE/config.yml:5]: One last suggestion and GTG, can we have our discord over rather than this? Thanks a lot

Comment (adityaraute) [reply .github/ISSUE_TEMPLATE/config.yml:5]: How do I link it? With an invite link? My invite may expire after 7 days.

Comment (xkrishguptaa) [reply .github/ISSUE_TEMPLATE/config.yml:5]: https://discord.eddiehub.org.

**Review by xkrishguptaa — APPROVED** [review]

**Pull request merged**